python/lsst/pex/config/config.py

```diff
@@ -414,30 +416,30 @@ def __iter__(self):
     def keys(self):
         """!Return the list of field names
         """
-        return self._storage.keys()
+        return list(self._storage.keys())
     def values(self):
         """!Return the list of field values
         """
-        return self._storage.values()
+        return list(self._storage.values())
     def items(self):
         """!Return the list of (field name, field value) pairs
         """
-        return self._storage.items()
+        return list(self._storage.items())
 
     def iteritems(self):
```

Comment:
There is the broader issue of what the `items` method should be returning now. It's a bit tricky because whilst we are in transition we can't change it to be what `iteritems` does.